lib/sufia/arkivo.rb

```diff
@@ -0,0 +1,11 @@
+require 'sufia/arkivo/config'
```

Comment:
@mjgiarlo Was there a decision to put this into lib instead of app?  It seems to be moving against the current trend